.vscode/extensions.json

```diff
@@ -0,0 +1,5 @@
+{
+    // See http://go.microsoft.com/fwlink/?LinkId=827846
+    // for the documentation about the extensions.json format
+    "recommendations": ["dbaeumer.vscode-eslint"]
```

Comment:
In my early days, I found the debugging experience annoying most of the time with all the library code so I never really use a debugger. Overall I am more productive with console logs in interactive applications.
So long story short. I don't know if a debugger works. Never used one for this project...

Reply:
I think the debuggers especially in VS Code got way better since then, I generally prefer using the debugger for debugging.
If it does not work automatically yet (which I doubt), I will put the recommendation in later myself.